### DISCUSSION TITLE
query empty date field

### DISCUSSION THREAD

**Discussion by Oggy4th** [pocketbase/pocketbase#2298] [Q&A] [answered]
`ShiftQuery(dao).AndWhere(dbx.HashExp{"endDate": nil}).Limit(1).One(shift)`

cannot select like this. Couldnt find any record.

How to select null DATE field?

**Comment by ganigeorgiev** [comment, marked as answer]
PocketBase uses zero-defaults and doesn't have `null` field values (with the exception of the `json` type field), aka. an empty date is empty string (`""`).

For a list with example supported values per field type you can check https://pocketbase.io/docs/collections/#overview.
